### PR TITLE
support testnet

### LINKED ISSUE #12: support testnet
Often developers do not have bsv, which is non-trivial to get. We wanna them to be able to start developing ASAP by getting test coins from a faucet.

### PR DESCRIPTION
**Pull request by zhfnjust**
Fix #12

### CODE REVIEW

**Comment by danwag06** [comment]
@zhfnjust What is the expectation for how this should work with the provider? Is the idea that the user (dev) flips the switch in their wallet and it sort of just handles everything? I assume passing mainet data via the provider while the switch is on testnet things would fail and vice versa... cc @xhliu

**Comment by zhfnjust** [comment]
what do you mean *the provider*, which provider?

**Comment by zhfnjust** [comment]
the user (dev) flips the switch, and it just handles everything.

**Comment by danwag06** [comment]
@zhfnjust when I refer to the "provider" I mean all of the functionality here:

https://panda-wallet.gitbook.io/provider-api/intro/introduction

File: src/components/ToggleSwitch.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import styled from "styled-components";
+import { ChangeEvent } from "react";
+import { Theme } from "../theme";
+
+const Label = styled.label`
+  display: flex;
+  align-items: center;
+  gap: 10px;
```
Comment (danwag06) [inline src/components/ToggleSwitch.tsx:8]: For future reference. For consistency, we'll use `rem` over `px` to match the other components. I've handled updating these is a separate branch.

**Review by danwag06 — APPROVED** [review]
PR looks good. Nice work 🚀

I've handled the design clean up and one small state change that was making the provider return outdated info based on the users switch setting in a separate branch 💪

**Pull request merged**